src/GeositeFramework/js/Map.js

```diff
@@ -101,20 +100,24 @@ require(['use!Geosite',
         view.model.on('change:extent', function () {
             var currentExtent = view.model.get('extent');
 
-            if (!_.isEqual(currentExtent, view.esriMap.extent)) {
+            if (!_.isEqual(currentExtent, view.esriMapView.extent)) {
                 loadExtent(view);
             }
         });
 
+        /*
         // Configure the esri proxy, for (at least) 2 cases:
         // 1) For WMS "GetCapabilities" requests
         // 2) When it needs to make an HTTP GET with a URL longer than 2000 chars
         esri.config.defaults.io.proxyUrl = "proxy.ashx";
-
+        */
         createMap(view);
     }
 
     function createMap(view) {
+        /*
+        // Reworking this because the Esri JS API 4.2
```

Comment:
Sounds good. I removed the nesting altogether from these comments. For commenting out single lines, I'll drop the `/*` & `*/` stuff and just prefix the line with `//`. For longer pieces, I'll keep the multiline comment format but move the `/*`s immediately before the disabled code.